### PR TITLE
Added the functions

### PR DESCRIPTION
**Pull request by GayathriManeksha**
#196

### CODE REVIEW

File: src/scrape_up/twitter/numidconverter.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.webdriver.chrome.options import Options
+from time import sleep
+from bs4 import BeautifulSoup
+import json
+
+def unametoid(username):
+    url = 'https://twitter.com/{}'.format(username)
```
Comment (nikhil25803) [inline src/scrape_up/twitter/numidconverter.py:10]: Hey @GayathriManeksha | The code-base looks good, but please follow the folder structure. 
+ Add `__init__.py` file in the `scrape_up/twitter` directory.
+ You can see the `scrape_up/twitter` directory for reference.
+ Also add the method and class in the `./documentation.md` file.

**Review by nikhil25803 — CHANGES_REQUESTED** [review]
Requested some changes @GayathriManeksha

**Comment by GayathriManeksha** [comment]
I made the changes mentioned

**Comment by nikhil25803** [comment]
Hey, there are some conflicts in the `documentation. md` file, do take a look into it.

**Comment by GayathriManeksha** [comment]
I have tried resolving the conflicts through the last commits. I guess I have done it right.

File: src/scrape_up/twitter/numidconverter.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    def idtouname(self,numid):
+        url = 'https://twitter.com/i/user/{}'.format(numid)
+        self.driver = webdriver.Chrome(options=self.chrome_options)
+        self.driver.get(url)
+        html = self.driver.page_source
+        soup = BeautifulSoup(html, 'html.parser')
+        user_id = soup.find('script', {'data-testid': 'UserProfileSchema-test'})
+        data = json.loads(user_id.string)
+        self.driver.quit()
+        return data['author']['additionalName']
```
Comment (nikhil25803) [inline src/scrape_up/twitter/numidconverter.py:45]: Please follow the return format of the project and use try-catch. It's a good practice ✌️
A proper method looks like this as shown in the code below.
image

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

File: src/scrape_up/twitter/numidconverter.py (hunk 2 above)
Comment (nikhil25803) [reply src/scrape_up/twitter/numidconverter.py:45]: Just a few more changes @GayathriManeksha, else looks good.

**Comment by GayathriManeksha** [comment]
I also tried resolving the conflicts again

**Comment by GayathriManeksha** [comment]
@nikhil25803 can you please review this?

**Review by nikhil25803 — APPROVED** [review]

**Comment by nikhil25803** [comment]
Looks good @GayathriManeksha 
Do not forget to 
+ Share your work on socials and tag us and Clueless 🫂
+ Give this repository a star ✨
+ Follow Clueless Community on GitHub ✅

**Pull request merged**